### PR TITLE
Fixed black dots on Tiberian Sun build palette icons

### PR DESCRIPTION
**Pull request by Mailaender**
Extracted from earlier versions of https://github.com/OpenRA/OpenRA/pull/8912.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Makes sense, I overlooked it could be defined there as well. Have to acknowledge that it does seem the most elegant way. :+1:

File: OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs [hunk 1]
```diff
@@ -46,6 +48,14 @@ public enum ReadyTextStyleOptions { Solid, AlternatingColor, Blinking }
 		public readonly string TooltipContainer;
 		public readonly string TooltipTemplate = "PRODUCTION_TOOLTIP";
 
+		public readonly string ClockAnimation = "clock";
+		public readonly string ClockSequence = "idle";
```
Comment (penev92) [inline OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs]: No SequenceReference ?

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs]: Add `desc`s please.

Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs]: These aren't traits.

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs]: Oh right, it isn't such a good idea to review from mobile.

**Comment by reaperrr** [comment]
:+1:

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/3c22226763a298fcfbe9d13d561622a77cc608fb...2e1622a96544e2f691ca5d6d824e051f2d632097)

**Pull request merged**